didcomm/protocols/forward/forward.py

```diff
@@ -24,11 +60,32 @@ class ForwardResult:
     forwarded_msg_encrypted_to: Optional[List[DID_URL]] = None
 
 
+# TODO move to a helper module
+async def find_did_service(
+    to_did_or_kid: DID_OR_DID_URL, resolvers_config: ResolversConfig
+) -> DIDCommService:
+
+    to_did = get_did(to_did_or_kid)
+    did_doc = await resolvers_config.did_resolver.resolve(to_did)
+
+    if did_doc is None:
+        raise DIDDocNotResolvedError()
+
+    # TODO do not restrict to the first only
+    try:
+        return did_doc.didcomm_services[0]
+    except IndexError:
+        raise DIDServiceNotFoundError
+
+
 async def wrap_in_forward(
     resolvers_config: ResolversConfig,
     packed_msg: Union[JSON_OBJ, JSON],
-    routing_key_ids: List[DID_OR_DID_URL],
-    forward_headers: Optional[List[Header]] = None,
+    # routing_key_ids: List[DID_OR_DID_URL],
+    to: DID_OR_DID_URL,
+    forward_headers: Optional[List[Header]] = None,  # TODO
+    pack_config: Optional[PackEncryptedConfig] = None,
+    pack_params: Optional[PackEncryptedParameters] = None,
```

Comment:
`pack_params` already contains `forward_headers`, doesn't it?

Reply:
let me check

Reply:
done